packages/@o3r/store-sync/src/sync-storage/storage-sync.ts

```diff
@@ -328,7 +332,7 @@ export const syncStorage = (config: SyncStorageConfig) => (reducer: any) => {
 
     // If state arrives undefined, we need to let it through the supplied reducer
     // in order to get a complete state as defined by user
-    if (action.type === INIT && !state) {
+    if (config.actionTypes?.includes(action.type) && !state) {
```

Comment:
```suggestion
    if (config.actionTypes!.includes(action.type) && !state) {
```

Because it can bot be null

Reply:
Done